_includes/scenarios/integrate/request-and-response-introduction.md

```diff
@@ -0,0 +1,49 @@
+- [FreeTextRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#freetextrequestitem)s can be used to send a free text to a contact, that in turn can accept this with a free text as well
+- [ProposeAttributeRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#proposeattributerequestitem)s can be used to ask the peer for Attributes, already suggesting answers, e.g. asking for an address and setting the country of the organization as default
+- [ReadAttributeRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#readattributerequestitem)s can be used to ask a peer for Attributes, e.g. asking for an address
+- [ShareAttributeRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#shareattributerequestitem)s can be used to share own IdentityAttributes with a contact, e.g. sharing the own DisplayName when requesting a new Relationship
+
+In case multiple RequestItems should be answered jointly, e.g. to enhance the structure for the user, you can combine them to a [RequestItemGroup]({% link _docs_integrate/data-model-overview.md %}#requestitemgroup).
+RequestItems or RequestItemGroups are then embedded in a Request, which can be sent to the other Identity.
+This happens either via a Template (see [Requests over Templates]({% link _docs_integrate/requests-over-templates.md %})) or via a Message (see [Requests over Messages]({% link _docs_integrate/requests-over-messages.md %})).
+
+Once the other Identity receives a Request, they can decide whether they want to accept or reject the corresponding RequestItems or RequestItemGroups.
+Consequently, the associated [ResponseItem]({% link _docs_integrate/data-model-overview.md %}#responseitem)s are created.
+They can either be [AcceptResponseItem]({% link _docs_integrate/data-model-overview.md %}#acceptresponseitem)s, [RejectResponseItem]({% link _docs_integrate/data-model-overview.md %}#rejectresponseitem)s or [ErrorResponseItem]({% link _docs_integrate/data-model-overview.md %}#errorresponseitem)s, in case the enmeshed Runtime detects a problem.
+Additionally, there are specific AcceptResponseItems extending the actual AcceptResponseItem to answer to RequestItems demanding additional information, like the ReadAttributeRequestItem.
+For every RequestItem or RequestItemGroup, there will be exactly one ResponseItem or [ResponseItemGroup]({% link _docs_integrate/data-model-overview.md %}#responseitemgroup) created, respectively.
+The [Response]({% link _docs_integrate/data-model-overview.md %}#response), then, is wrapped in a [ResponseWrapper]({% link _docs_integrate/data-model-overview.md %}#responsewrapper), which holds additional information about the corresponding Request, so that it can be processed correctly once it is returned.
+
+## Example: Request-Response-Process for a CreateAttributeRequestItem
```

Comment:
This example is quite a complex one. What about have a AuthenticationRequestItem or FreeTextRequestItem instead?
Additionally, there are some misunderstandings in how it works internally ;) But I guess this should be an intro and the internal stuff should be moved to a more in-depth documentation.
I would love to see more diagrams and different examples to grasp the overview, and not the technical details of a specific scenario.